test/node.js

```diff
@@ -24,5 +24,12 @@ exports.node = {
     Cookies.remove('anything')
     Cookies.remove('anything', { path: '' })
     test.done()
+  },
+  should_expose_rfc_6265_converter: function (test) {
```

Comment:
I noticed that we were using snakecase here (might have been added by a pull request and we didn't pay attention) and now changed that: 9834ceb0de76d29629d6c69c4611587326e4043d

Could you adapt this? Hat tip for noticing and keeping it consistent in here 🙇